src/test/java/edu/ucsb/ucsbcslas/controllers/CourseControllerTests.java

```diff
@@ -245,81 +243,4 @@ public void testDeleteCourse_courseNotFound() throws Exception {
-  }
-
-  @Test
-  public void testGetMyCourses_notAdmin() throws Exception {
-    List<Course> expectedCourses = new ArrayList<Course>();
-    expectedCourses.add(new Course(1L, "course 1", "F20", "fname", "lname", "email"));
-    when(mockCourseRepository.findAllByInstructorEmail("email")).thenReturn(expectedCourses);
-    when(mockAuthControllerAdvice.getIsAdmin(anyString())).thenReturn(false);
-    AppUser user = new AppUser(1L, "email", "Seth", "VanB");
-    when(mockAuthControllerAdvice.getUser(anyString())).thenReturn(user);
-    MvcResult response = mockMvc.perform(get("/api/member/courses").contentType("application/json")
-        .header(HttpHeaders.AUTHORIZATION, "Bearer " + userToken())).andExpect(status().isOk()).andReturn();
-
-    verify(mockCourseRepository, times(1)).findAllByInstructorEmail("email");
-
-    String responseString = response.getResponse().getContentAsString();
-    List<Course> actualCourses = objectMapper.readValue(responseString, new TypeReference<List<Course>>() {
-    });
-    assertEquals(actualCourses, expectedCourses);
-  }
-
-
-  @Test
-  public void testGetMyCoursesUnauthorized() throws Exception {
-    List<Course> expectedCourses = new ArrayList<Course>();
-    when(mockCourseRepository.findAllByInstructorEmail("email")).thenReturn(expectedCourses);
-    when(mockAuthControllerAdvice.getIsAdmin(anyString())).thenReturn(false);
-    AppUser user = new AppUser(1L, "email", "Seth", "VanB");
-    when(mockAuthControllerAdvice.getUser(anyString())).thenReturn(user);
-    mockMvc.perform(get("/api/member/courses").contentType("application/json")
-        .header(HttpHeaders.AUTHORIZATION, "Bearer " + userToken())).andExpect(status().isUnauthorized());
-  }
-
-  @Test
-  public void testGetMyCoursesInstructor() throws Exception {
-    List<Course> expectedCourses = new ArrayList<Course>();
-    expectedCourses.add(new Course(1L, "course 1", "F20", "fname", "lname", "email"));
-    when(mockCourseRepository.findAllByInstructorEmail("email")).thenReturn(expectedCourses);
-    when(mockAuthControllerAdvice.getIsMember(anyString())).thenReturn(true);
-    MvcResult response = mockMvc.perform(get("/api/member/courses/forInstructor/email").contentType("application/json")
-        .header(HttpHeaders.AUTHORIZATION, "Bearer " + userToken())).andExpect(status().isOk()).andReturn();
-
-    verify(mockCourseRepository, times(1)).findAllByInstructorEmail("email");
-
-    String responseString = response.getResponse().getContentAsString();
-    List<Course> actualCourses = objectMapper.readValue(responseString, new TypeReference<List<Course>>() {
-    });
-    assertEquals(actualCourses, expectedCourses);
-  }
-
-  @Test
-  public void testGetMyCoursesInstructorUnauthorized() throws Exception {
-    mockMvc.perform(get("/api/member/courses/forInstructor/email").contentType("application/json")
-    .header(HttpHeaders.AUTHORIZATION, "Bearer " + userToken())).andExpect(status().isUnauthorized());
-  }
```

Comment:
Like the comment above in `CourseController.java`, it appears that you're deleting code that another team put in. We don't want to undo the work of other teams, so can you please put this back in?

Reply:
I added them back in